data/strings/types_strings.txt

```diff
@@ -19802,7 +19802,7 @@
     fa = دفتر بیمه
     fi = Vakuutuslaitos
     fr = Bureau d’assurance
-    hu = Biztosító iroda
+    hu = Biztosítóiroda
```

Comment:
It was correct as it was separated
![Képernyőkép ekkor: 2024-02-24 13-12-38](https://github.com/organicmaps/organicmaps/assets/76624594/b2af97bc-9b35-4d9e-8f2e-6660b3d22145)

Reply:
Itt a helyesiras.mta.hu mást mond: ```
A „biztosító” melléknevet és az „iroda” főnevet egybeírjuk az alábbi szabály alapján:
Ha arról van szó, hogy a(z) „biztosító” valamilyen képességre, rendeltetésre vonatkozik, a(z) iroda nem vagy nem pusztán pillanatnyi cselekvést, tevékenységet végez, illetve folyamatot átél, elszenved, akkor a kifejezést egybe kell írni. [[AkH11-112](https://helyesiras.mta.hu/helyesiras/default/akh11#112), [AkH12-118](https://helyesiras.mta.hu/helyesiras/default/akh12#118), OH-113]
```